content/participants/ttcj@viral-sangani/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Viral Sangani
+type: "Participant"
+name: "Viral Sangani"
+portfolio: ""
+github-link: "https://github.com/viral-sangani"
+linkedin-link: "https://linkedin.com/in/tanaypratap"
```

Comment:
Please use your own LinkedIn link 🔢

Reply:
Please review it again, I have made the required changes.